package.json

```diff
@@ -3,9 +3,10 @@
   "private": true,
   "description": "package.json intended for in-repo use only, package.json used for publishing is located in src/AutoRest.CSharp/package.json",
   "devDependencies": {
-    "@azure-tools/typespec-csharp": "src/TypeSpec.Extension/Emitter.Csharp",
     "@azure-tools/cadl-ranch-mockapis": "file:test/CadlRanchMockApis",
+    "@azure-tools/typespec-csharp": "src/TypeSpec.Extension/Emitter.Csharp",
     "@microsoft.azure/autorest.testserver": "3.3.24",
+    "@types/mocha": "^10.0.1",
```

Comment:
autorest.csharp does not depend on `@types/mocha`. `Emitter.Csharp` depends on it which is already his package.json. 
We may remove this.

Reply:
Sure, I will revert this change